projects/cyclegan_reference_simulations/README.md

```diff
@@ -0,0 +1,132 @@
+# CycleGAN reference simulations
+
+This directory contains configuration information and an example runscript for
+the reference simulations used in McGibbon et al. (2023).  These simulations
+were completed using the following versions of
+[fv3net](https://github.com/ai2cm/fv3net) and
+[fv3gfs-fortran](https://github.com/ai2cm/fv3gfs-fortran) on various partitions
+of GFDL's Gaea computer.
+
+- fv3net:
+  `1d0967c5c7f7c347b25ecf7a2ce9ebd363e848ea`
+- fv3gfs-fortran:
+  `0a446035cc4730b9319a6925a438a22063117c7e`
+
+The fortran executable and Python environment used to run these simulations can
+be built on Gaea following the instructions
+[here](https://github.com/ai2cm/fv3net/tree/master/.environment-scripts#building-an-environment-on-gaea).
+
+## Configuration files
+
+Configuration YAML files can be found in the `configurations` directory.  These
+are defined for each resolution, C48 or C384, and climate (-2 K, 0 K, +2 K, +4
+K, or ramped).  These YAML files are meant for use with
+[fv3config](https://github.com/ai2cm/fv3config), which is a Python package
```

Comment:
nit: Could specify version here, though it will already be pinned in the zenodo archive from the top level.